lib/virtio/virtqueue.c

```diff
@@ -56,19 +56,21 @@ static inline metal_phys_addr_t virtqueue_virt_to_phys(struct virtqueue *vq,
 }
 
 /**
- * virtqueue_create - Creates new VirtIO queue
- *
- * @param device    - Pointer to VirtIO device
- * @param id        - VirtIO queue ID , must be unique
- * @param name      - Name of VirtIO queue
- * @param ring      - Pointer to vring_alloc_info control block
- * @param callback  - Pointer to callback function, invoked
- *                    when message is available on VirtIO queue
- * @param notify    - Pointer to notify function, used to notify
- *                    other side that there is job available for it
- * @param vq        - Created VirtIO queue.
- *
- * @return          - Function status
+ * @internal
+ *
+ * @brief Creates new VirtIO queue
+ *
+ * @param device	Pointer to VirtIO device
+ * @param id		VirtIO queue ID , must be unique
+ * @param name		Name of VirtIO queue
+ * @param ring		Pointer to vring_alloc_info control block
+ * @param callback	Pointer to callback function, invoked
+ *			when message is available on VirtIO queue
+ * @param notify	Pointer to notify function, used to notify
+ *			other side that there is job available for it
+ * @param vq		Created VirtIO queue.
+ *
+ * @return Function status
```

Comment:
If you have possibility to move the function headers in the virqueue.h, I would appreciate. But this is not mandatory for this PR, can be done later.